### PR TITLE
Completed MVP and Stretch challenges

### PR DESCRIPTION
**Pull request by mngmay**
@ayunas Answered questions in all files.

### CODE REVIEW

File: assignments/arrays.js [hunk 1]
```diff
@@ -63,32 +63,42 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
-console.log(); 
+let oldCars = carYears.filter(function(car){
+    if(car < 2000){ 
+        return car}});
+
+console.log(oldCars.length); 
 
 // ==== Challenge 6 ====
 // A buyer is interested in seeing only BMW and Audi cars within the inventory.  Return an array that only contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
-let BMWAndAudi = [];
```
Comment (ayunas) [inline assignments/arrays.js:90]: you can't declare a let variable more than once.  every place you did that, it's generating an error.

File: assignments/function-conversion.js [hunk 2]
```diff
@@ -5,26 +5,49 @@
...
 
+let myFunction = () => console.log("Function was invoked!");
+myFunction();
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => {return param};
```
Comment (ayunas) [inline assignments/function-conversion.js:16]: on a single line, you can take out the brackets and the return keyword.

File: assignments/function-conversion.js [hunk 3]
```diff
@@ -5,26 +5,49 @@
...
 // exampleArray = [1,2,3,4];
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
+
+exampleArray = [1,2,3,4];
+const triple = exampleArray.map((num) => {
```
Comment (ayunas) [inline assignments/function-conversion.js:50]: with a single argument, the parenthesis in an arrow function are not required.

File: assignments/objects.js [hunk 4]
```diff
@@ -49,16 +97,42 @@ const example = {
...
 // Have the parent speak
 
+parent.speak();
+
 // Have the child speak
 
+parent.child.speak();
+
 // Have the grandchild speak
+
```
Comment (ayunas) [inline assignments/objects.js:137]: looking good. just a few syntax errors

**Review by ayunas — APPROVED** [review]

**Comment by ayunas** [comment]
@mngmay everything looks good.  the major thing is you cannot declare let variables more than once.  once you declare it one time, then use it directly without writing let a second time.

File: assignments/arrays.js (hunk 1 above)
Comment (mngmay) [reply assignments/arrays.js]: Hi Amir, I only have one instance of let for each variable in my actual file.  I think this is showing that I overwrote the existing provided variable with the function that I wrote.

Comment (mngmay) [reply assignments/arrays.js:90]: Hi Amir, my file only contains the declarations in green. The red declaration is the original code provided in the file which was overwritten by my solutions.  When running the file, these do not generate an error for me.

**Pull request merged**